python/custreamz/custreamz/kafka.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+import confluent_kafka as ck
+from cudf_kafka._lib.kafka import KafkaDatasource
+
+import cudf
+
+from custreamz.utils import docutils
+
+
+# Base class for anything class that needs to interact with Apache Kafka
+class CudfKafkaClient(object):
+    def __init__(self, kafka_configs):
```

Comment:
Can you add a docstring to the `__init__` method here describing the class and its initialization?